src/components/Budget/BudgetView/BudgetViewContainer.js

```diff
@@ -74,6 +82,32 @@ const BudgetViewContainer = ({ history, match, mutator }) => {
     [history, budget],
   );
 
+  const removeBudget = useCallback(
+    () => {
+      mutator.budgetById.DELETE({ id: budgetId })
+        .then(() => {
+          showCallout({ messageId: 'ui-finance.budget.actions.remove.success', type: 'success' });
+          history.replace({
```

Comment:
why it's replace?

Reply:
to block redirecting back to invalid URL since the budget is deleted